stories/datafusion.stories.js

```diff
@@ -176,3 +176,121 @@ export const getMapProcessingEvalscriptUrl = () => {
 
   return wrapperEl;
 };
+
+export const getMapProcessingGainGamma = () => {
+  if (!process.env.CLIENT_ID || !process.env.CLIENT_SECRET) {
+    return "<div>Please set OAuth Client's id and secret for Processing API (CLIENT_ID, CLIENT_SECRET env vars)</div>";
+  }
+
+  const imgNoGainGamma = document.createElement('img');
+  imgNoGainGamma.width = '256';
+  imgNoGainGamma.height = '256';
+
+  const imgGainIs2 = document.createElement('img');
+  imgGainIs2.width = '256';
+  imgGainIs2.height = '256';
+
+  const imgGammaIs2 = document.createElement('img');
+  imgGammaIs2.width = '256';
+  imgGammaIs2.height = '256';
+
+  const imgGainGammaAre2 = document.createElement('img');
+  imgGainGammaAre2.width = '256';
+  imgGainGammaAre2.height = '256';
+
+  const wrapperEl = document.createElement('div');
+  wrapperEl.innerHTML = '<h2>GetMap with Processing and gain and gamma effects</h2>';
+  wrapperEl.innerHTML += '<h4>no gain/gamma | gain | gamma | gain and gamma</h4>';
+  wrapperEl.insertAdjacentElement('beforeend', imgNoGainGamma);
+  wrapperEl.insertAdjacentElement('beforeend', imgGainIs2);
+  wrapperEl.insertAdjacentElement('beforeend', imgGammaIs2);
+  wrapperEl.insertAdjacentElement('beforeend', imgGainGammaAre2);
+
+  const perform = async () => {
+    await setAuthTokenWithOAuthCredentials();
+
+    const layerS2L1C = new S2L1CLayer({ instanceId, layerId: s2l1cLayerId });
+    const layerS2L2A = new S2L2ALayer({
+      instanceId,
+      layerId: s2l2aLayerId,
+      mosaickingOrder: MosaickingOrder.LEAST_RECENT,
+    });
+    const layers = [
+      {
+        layer: layerS2L2A,
+        id: 's2l2a',
+        fromTime: new Date(Date.UTC(2020, 1 - 1, 1, 0, 0, 0)),
+        toTime: new Date(Date.UTC(2020, 2 - 1, 26, 0, 0, 0)),
+      },
+      {
+        layer: layerS2L1C,
+        id: 's2l1c',
+        fromTime: new Date(Date.UTC(2020, 2 - 1, 10, 0, 0, 0)),
+        toTime: new Date(Date.UTC(2020, 2 - 1, 24, 0, 0, 0)),
+      },
+    ];
+    const layer = new ProcessingDataFusionLayer({
+      layers: layers,
+      evalscript: `
+        //VERSION=3
+        var setup = () => ({
+          input: [
+            {datasource: "s2l2a", bands:["B02", "B03", "B04"], units: "REFLECTANCE", mosaicking: "ORBIT"},
+            {datasource: "s2l1c", bands:["B02", "B03", "B04"], units: "REFLECTANCE"}],
+          output: [
+            {id: "default", bands: 3, sampleType: SampleType.AUTO}
+          ]
+        });
+
+
+        function evaluatePixel(samples, inputData, inputMetadata, customData, outputMetadata) {
+          var sample = samples.s2l2a[0];
+          if (!sample) {
+            return {
+              default: [0, 0, 0],
+            }
+          }
+          let val = [sample.B04, sample.B03, sample.B02];
+
+          return {
+            default: val
+          }
+        }
+      `,
```

Comment:
Thanks, that will look nicer. I just copied the evalscript from some other story because I didn't want to waste time looking if some datafusion custom script already exists in custom script repo.